src/ansys/aedt/core/modeler/cad/primitives.py

```diff
@@ -3213,7 +3213,13 @@ def sweep_along_path(
 
         self.oeditor.SweepAlongPath(vArg1, vArg2)
 
-        return self.update_object(assignment)
+        if isinstance(assignment, list):
+            updated_obj = []
+            for sel_obj in assignment:
+                updated_obj.append(self.update_object(sel_obj))
+            return updated_obj
```

Comment:
```suggestion
            res = []
            for sel_obj in assignment:
                res.append(self.update_object(sel_obj))
            return res
```
Just a minor refactor to use a "standard" name for return value.